src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java

```diff
@@ -123,16 +121,33 @@ public BeachfrontBidder(String bannerEndpointUrl, String videoEndpointUrl) {
         final BeachfrontBannerRequest beachfrontBannerRequest = makeBannerRequest(bidRequest, bidRequest.getImp(),
                 errors);
         return Result.of(Collections.singletonList(
-                HttpRequest.<BeachfrontRequests>builder()
+                getHttpRequestBuilderWithDeviceHeaders(bidRequest.getDevice())
                         .method(HttpMethod.POST)
                         .uri(endpoint)
                         .body(Json.encode(beachfrontBannerRequest))
-                        .headers(BidderUtil.headers())
                         .payload(BeachfrontRequests.of(beachfrontBannerRequest, null))
                         .build()),
                 makeBadInputErrors(errors));
     }
 
+    //Also can include body as Object parameter, and set method POST by defoult
+    private static HttpRequestBuilder<BeachfrontRequests> getHttpRequestBuilderWithDeviceHeaders(Device device) {
```

Comment:
Rename method name to `httpRequestBuilderWithPopulatedHeadersAndMethod()`.
Yes, i think we can add `HttpMethod httpMethod` to method signature, since it is the same for both banner & video requests.